sympy/concrete/tests/test_sums_products.py

```diff
@@ -934,6 +934,7 @@ def test_is_convergent():
     assert Sum(1/(n**2 + 1), (n, 1, oo)).is_convergent() is S.true
     assert Sum(1/n**(S(6)/5), (n, 1, oo)).is_convergent() is S.true
     assert Sum(2/(n*sqrt(n - 1)), (n, 2, oo)).is_convergent() is S.true
+    assert Sum(1/(sqrt(n)*sqrt(n)), (n, 2, oo)).is_convergent() is S.false
```

Comment:
I wonder as to why this wasn't simply written as `1 / (sqrt(n) * sqrt(n))` and not  `1 / n` ?

Reply:
Probably to make it less obvious. Something like `1/sqrt(n) * sqrt(n+1))` might have been better, but I think we've established that it's impossible to find a series that would by detected with this addition and not otherwise.